src/open_inwoner/openzaak/tests/test_case_detail.py

```diff
@@ -2081,3 +2230,68 @@ def test_request_error_in_connecting_doc_with_zaak_shows_proper_message(
                 f"Een fout is opgetreden bij het uploaden van {self.uploaded_informatie_object['bestandsnaam']}"
             ),
         )
+
+    def test_kcm_redirect(self, m):
+        """Check redirect from question embedded in case detail to klant_contactmoment detail"""
+
+        self._setUpMocks(m)
```

Comment:
Yes, the test needs the `contactmoment` mock, which is also used elsewhere in the suite. I'm defining additional mocks inside the test only if they aren't needed anywhere else. At first I simply added these new mocks to `setUp`, which would install them for every test, but I got complaints about missing mock addresses.

I thought it would make sense to install these extra mocks only where they're needed, but if it's poor style I can move them back into the general `setUp` function.